frontend/packages/dev-console/src/components/import/advanced/ResourceLimitSection.tsx

```diff
@@ -5,15 +5,37 @@ import { useFormikContext, FormikValues } from 'formik';
 import FormSection from '../section/FormSection';
 import { MemoryUnits, CPUUnits } from '../import-types';
 
-const ResourceLimitSection: React.FC = () => {
+export type ResourceLimitSectionProps = {
+  hideTitle?: boolean;
+  showHelpText?: boolean;
+  container?: React.ReactNode;
+};
+
+const ResourceLimitSection: React.FC<ResourceLimitSectionProps> = ({
+  hideTitle,
+  showHelpText,
+  container,
+}) => {
   const { t } = useTranslation();
   const {
     values: {
       limits: { cpu, memory },
     },
   } = useFormikContext<FormikValues>();
   return (
-    <FormSection title={t('devconsole~Resource limit')}>
+    <FormSection title={!hideTitle && t('devconsole~Resource limit')} fullWidth>
+      {showHelpText && (
+        <p>
+          {t(
+            'devconsole~Resource limits control how much CPU and memory a container will consume on a node.',
+          )}
+        </p>
+      )}
```

Comment:
```suggestion
    <FormSection title={!hideTitle && t('devconsole~Resource limit')} helpText= {t(
            'devconsole~Resource limits control how much CPU and memory a container will consume on a node.',
          )} fullWidth>
      
```

Reply:
This subtitle is to be shown only in the modal view, so I have added a check

Reply:
I think its fine to show help text for resource limits section in other forms as well(import from git, deploy image etc). Also even if we want to conditionally show helpText it can be simply done like this:
```
<FormSection title={!hideTitle && t('devconsole~Resource limit')} helpText= {showHelpText && t('devconsole~Resource 
  limits control how much CPU and memory a container will consume on a node.',)} fullWidth>
```
But we dont need to do this since its alright to show helpText in all forms.